code/obj/item.dm

```diff
@@ -1254,6 +1254,9 @@
 	if(user.is_hulk())
 		power *= 1.5
 
+	var/datum/limb/attacking_limb = user?.equipped_limb()
```

Comment:
while I do like this as a solution, I did a quick look and it looks like it gets overridden a lot. Honestly this proc feels like it ought to have `SHOULD_CALL_PARENT` set on it, but it'd probably be a huge job to do that.

Reply:
Most if not all of the overrides are for unique stuff like using ampoules I think, in any place you wouldn't want to do damage